.github/workflows/pr.yml

```diff
@@ -0,0 +1,29 @@
+name: PR
```

Comment:
Nice, it works!

https://github.com/Skyscanner/babel-plugin-transform-i18n/actions/runs/5912985878/job/16036919899?pr=14